backend/backend.go

```diff
@@ -65,6 +65,7 @@ var fixedURLWhitelist = []string{
 	"https://ext.shiftcrypto.ch/",
 	"https://shop.shiftcrypto.ch/",
```

Comment:
Since you're here, please replace this with `https://shiftcrypto.shop/`.

Reply:
done